widgets/common/src/ManageLabelsModal.jsx

```diff
@@ -73,11 +84,18 @@ function ManageLabelsModal({ deploymentId, open, onHide, toolbox }) {
 
 ManageLabelsModal.propTypes = {
     deploymentId: PropTypes.string.isRequired,
+    existingLabels: PropTypes.arrayOf(),
```

Comment:
That should probably be extended to:
```suggestion
    existingLabels: PropTypes.arrayOf(PropTypes.string),
```
?

Reply:
I reused LabelsPropType here.